src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -275,18 +295,25 @@ export class HuiCardPicker extends LitElement {
         >
           ${!element || element.tagName === "HUI-ERROR-CARD"
             ? html`
-                ${this.hass!.localize(
-                  `ui.panel.lovelace.editor.card.${cardConfig.type}.description`
-                )}
+                ${customCard
+                  ? customCard.description ||
+                    this.hass!.localize(
+                      `ui.panel.lovelace.editor.card.custom.description`
+                    )
```

Comment:
We should just render `customCard.type` instead of a generic message.

Reply:
Or I guess just nothing at all ? The text "A custom card" doesn't feel right.

Reply:
Maybe "No description available" ?

Reply:
Also think that the localize key should be part of the card picker translation strings, so in the future we can conditionally load it with just the dialog.

Reply:
... I don't understand that last one...